ultravox/model/ultravox_processing.py

```diff
@@ -89,6 +90,7 @@ def __call__(
         text: Optional[str] = None,
         audio: Optional[Union[np.ndarray, torch.Tensor]] = None,
         sampling_rate: Optional[int] = None,
+        audio_context_size: Optional[int] = None,
```

Comment:
I would probably put `audio_context_size` in the processor `__init__` rather than `__call__` because it's not expected to change with each sample.

Also I feel like `UltravoxProcessor` class should come up with the right defaults on its own, otherwise you'd have to specify it in too many places: 1) inference (you already did)  2) training  3) when someone else loads UltravoxPipeline via `.from_pretrained`.